### PR TITLE
zh-CN: SUMMARY translation

### PR DESCRIPTION
**Pull request by mobyw**
https://github.com/google/comprehensive-rust/issues/324

### CODE REVIEW

File: po/zh-CN.po [hunk 1]
```diff
@@ -50,14 +50,13 @@ msgstr "在本地运行 Cargo"
...
+msgstr "第一天：上午"
 
 #: src/SUMMARY.md
 msgid "Welcome"
 msgstr "欢迎"
 
 #: src/SUMMARY.md src/hello-world.md src/hello-world/hello-world.md
-#, fuzzy
 msgid "Hello, World"
 msgstr "Hello World!"
```
Comment (henrif75) [inline po/zh-CN.po:61]:
```suggestion
msgstr "Hello, World"
```

**Review by henrif75 — CHANGES_REQUESTED** [review]

File: po/zh-CN.po [hunk 2]
```diff
@@ -259,27 +241,24 @@ msgid "Traits"
...
 
 #: src/SUMMARY.md src/generics.md
 msgid "Generics"
 msgstr "泛型"
 
 #: src/SUMMARY.md src/generics/generic-functions.md
-#, fuzzy
 msgid "Generic Functions"
-msgstr "外部函数"
+msgstr "泛型函数"
```
Comment (AgainstEntropy) [inline po/zh-CN.po:257]: Sorry I should have use suggestion
I think “泛型函数” will it be more consistent than "泛化函数"

File: po/zh-CN.po [hunk 3]
```diff
@@ -1055,19 +951,19 @@ msgstr "编写 UART 驱动程序"
 
 #: src/SUMMARY.md
 msgid "More Traits"
-msgstr "更多 trait"
+msgstr "更多 Trait"
```
Comment (AgainstEntropy) [inline po/zh-CN.po:954]:
```suggestion
msgstr "更多特征"
```

File: po/zh-CN.po [hunk 4]
```diff
@@ -237,16 +221,14 @@ msgid "Day 2: Morning"
 msgstr "第二天：上午"
 
 #: src/SUMMARY.md src/pattern-matching/let-control-flow.md
-#, fuzzy
 msgid "Let Control Flow"
-msgstr "控制流"
+msgstr "控制流结构"
```
Comment (AgainstEntropy) [inline po/zh-CN.po:225]:
```suggestion
msgstr "`let` 控制流"
```

File: po/zh-CN.po [hunk 5]
```diff
@@ -389,12 +358,11 @@ msgstr "内存管理"
 
 #: src/SUMMARY.md src/memory-management/review.md
 msgid "Review of Program Memory"
-msgstr "对程序内存进行审查"
+msgstr "程序的内存分配"
```
Comment (AgainstEntropy) [inline po/zh-CN.po:361]:
```suggestion
msgstr "回顾：程序的内存分配"
```

**Review by AgainstEntropy — APPROVED** [review]

**Review by henrif75 — APPROVED** [review]

**Comment by henrif75** [comment]
Thanks!

**Pull request merged**